docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.inputMapper.base`
+
+<!-- TODO: Add links to the inputMapper component's documentation -->
+
+This component transforms the gamepad inputs into actions. It is derived from the combination of two grades - the
+[`navigator`](navigator.md) grade, which reads and stores the gamepad inputs in the form of model data, and the
+[`configMaps`](configMaps.md) grade provides a configuration map for the gamepad inputs. The navigation features
```

Comment:
This should probably match the first part of the sentence, i.e. "grade, which provides".